src/api/app/components/notification_avatars_component.html.haml

```diff
@@ -0,0 +1,16 @@
+%ul.list-inline.d-flex.flex-row-reverse.avatars.m-0
+  - if avatar_objects.size > 6
+    %li.list-inline-item
+      - hidden = (avatar_objects.size - 6)
```

Comment:
What about moving this logic into the component? That way you can document those `6` and present a simpler interface in the view. :thought_balloon:

Reply:
I added a `NotificationAvatarsComponent` for this. I also refactored the code a bit, moving logic from the view to the component and using a constant instead of `6`, among others.